packages/js-sdk/src/loadContracts.d.ts

```diff
@@ -1,8 +1,8 @@
-import { ethers, Contract as EthersContract, utils } from "ethers"; 
+import type { ethers, Contract as EthersContract, utils } from "ethers"; 
 import type { Contract as Web3Contract } from "web3-eth-contract";
 import TruffleContract from "@truffle/contract";
 import type Web3 from "web3";
-import type { Contract } from 'ethers'
+import type { Contract, ContractInterface } from 'ethers'
```

Comment:
One small comment: you are already importing ethers above and giving `Contract` an alias of `EthersContract`, I think it makes sense to remove `Contract` and just use `EthersContract` on line 27 and to move `ContractInterface` into the import in line 1.
Everything else looks great though!